testkit-backend/pom.xml

```diff
@@ -35,8 +35,6 @@
         <dependency>
             <groupId>org.projectlombok</groupId>
             <artifactId>lombok</artifactId>
-            <version>1.18.20</version>
-            <scope>provided</scope>
```

Comment:
personal preference again: I usually let the scope visible

Reply:
The `svm` dependency is currently used in the same way. I feel like we should keep it consistent or have a separate PR to update them both.